docs/migrations/25-04.md

```diff
@@ -37,6 +37,35 @@ The third preview of workflow outputs introduces the following breaking changes
 
 See {ref}`workflow-output-def` to learn more about the workflow output definition.
 
+<h3>Workflow params (first preview)</h3>
+
+The `params` block is a new way to declare pipeline parameters in a Nextflow script:
+
+```nextflow
+nextflow.preview.params = true
+
+params {
+  /**
+   * Path to input data.
+   */
+  input
```

Comment:
it should not be allowed the use of undefined values

Reply:
This is how you specify a required parameter. When there is no default value and you don't specify a value at runtime, the run will fail